eslzArm/managementGroupTemplates/policyAssignments/ENFORCE-LZDecommissionedPolicyAssignment.json

```diff
@@ -0,0 +1,59 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-08-01/managementGroupDeploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "topLevelManagementGroupPrefix": {
+            "type": "string",
+            "metadata": {
+                "description": "Provide the ESLZ company prefix to the intermediate root management group containing the policy definitions."
+            }
+        },
+        "enforcementMode": {
+            "type": "string",
+            "allowedValues": [
+                "Default",
+                "DoNotEnforce"
+            ],
+            "defaultValue": "Default"
+        },
+        "listOfResourceTypesAllowed": {
+            "type": "Array",
+            "defaultValue": [
+                "microsoft.consumption/tags"
```

Comment:
Thinking we should also add to the allow list here.

- Role Assignments
    - Also PIM role assignments 
- Role Definitions
- Policy Assignments
- Resoruce Locks
- Policy Defs/Initiatives 
- Microsoft.Resources/tags

This will allow platform teams to at least start clearing stuff up or adjusting RBAC as needed etc.